lib/hexo/index.js

```diff
@@ -317,8 +317,9 @@ class Hexo extends EventEmitter {
     const { cache } = Object.assign({
       cache: false
     }, this.config.server);
+    const { alias } = this.extend.console;
 
-    if (this.env.cmd.startsWith('s') && cache) {
+    if (Object.keys(alias).filter(key => alias[key] === 'server').includes(this.env.cmd) && cache) {
```

Comment:
```suggestion
    if (alias[this.env.cmd] === 'server' && cache) {
```

It is much simpler, and could do the same job.

Reply:
👍